src/odemis/util/spot.py

```diff
@@ -131,7 +134,25 @@ def SpotIntensity(data, background=None):
 #     return [FindCenterCoordinates(i) for i in subimages]
 
 
-def FindCenterCoordinates(image, smoothing=True):
+def FindCenterCoordinates(
+    image: numpy.ndarray, smoothing: bool = True
+) -> Tuple[float, float]:
+    """
+    Returns the radial symmetry center of the image with sub-pixel resolution.
+    The position is returned as a tuple `(dx, -dy)` in pixels relative to the
+    center of the image.
+
+    For more information see radial_symmetry_center().
+
+    """
+    ji = radial_symmetry_center(image, smoothing)
+    xc, yc = to_physical_space(ji, image.shape)
```

Comment:
Is `to_physical_space` now only used to move the origin to the center of the image?

Reply:
The idea here is to keep `FindCenterCoordinates()` for backwards compatibility with existing functions.

The code shows that `radial_symmetry_center()` returns values in pixel coordinates. And that `FindCenterCoordinates` returns in a confusing `x, -y` reference system.